src/web/mjs/connectors/Piccoma.mjs

```diff
@@ -7,7 +7,7 @@ export default class Piccoma extends Connector {
         super();
         super.id = 'piccoma';
         super.label = 'Piccoma';
-        this.tags = ['webtoon', 'japanese'];
+        this.tags = ['manga', 'webtoon', 'japanese'];
         this.url = 'https://piccoma.com/web';
```

Comment:
https://jp.piccoma.com/web/ works for me, not this one.

Reply:
This fix does not pertain to that issue. By the way, I have fixed that issue. (Resolves #5231)